src/seedu/addressbook/data/AddressBook.java

```diff
@@ -74,4 +74,8 @@ public boolean equals(Object other) {
                 || (other instanceof AddressBook // instanceof handles nulls
                         && this.allPersons.equals(((AddressBook) other).allPersons));
     }
+    private  static void helloworld()
```

Comment:
Please leave a newline between functions. Also, there's a double space between your `private` and `static`